src/OrchardCore/OrchardCore.Search.Elasticsearch.Core/Services/ElasticIndexingService.cs

```diff
@@ -100,7 +100,7 @@ public async Task ProcessContentItemsAsync(params string[] indexNames)
             }
 
             IEnumerable<IndexingTask> tasks = [];
-            var allContentTypes = indexSettingsList.SelectMany(x => x.IndexedContentTypes ?? []).Distinct().ToList();
```

Comment:
@Skrypt I actually let this one here on purpose. Maybe use `.ToArray()` for better optimization. If we don't cast it here, we'll reevaluate it over and over in the loop. So it is best to allocate the result once to optimize the code.

Reply:
Yeah, it's an array of strings. .ToArray then

Reply:
I'll get that added back. Let me know if you want anything else done here or approve please

Reply:
@Skrypt I added it back. Let me know if you see anything else.